index.js

```diff
@@ -16,8 +16,8 @@ var cookies = new Cookies()
 follow(process.argv[2], start)
 
 function setCookie (url, headers) {
-  var cookieHeader = headers['set-cookie']
-  if (!cookieHeader) return 0
+  if (!headers['set-cookie']) return 0
+  var cookieHeader = [].concat(headers['set-cookie'])
```

Comment:
Yeah, just checked the http2 source code and you're right - It will not be an array if there's only one `Set-Cookie` header. So let's keep your implementation as is. I've sent a pull request (molnarg/node-http2#179) to http2 trying to fix this. But there's no need to wait for that.